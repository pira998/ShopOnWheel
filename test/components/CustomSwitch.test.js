import React from 'react';
import renderer from 'react-test-renderer';
import CustomSwitch from '../../components/CustomSwitch'


test('renders correctly', () => {
  const tree = renderer.create(<CustomSwitch />).toJSON();
  expect(tree).toMatchSnapshot();
});