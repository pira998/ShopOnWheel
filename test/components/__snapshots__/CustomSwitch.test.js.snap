// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "borderColor": "#898B9A",
        "borderRadius": 10,
        "borderWidth": 1,
        "height": 20,
        "justifyContent": "center",
        "paddingLeft": 2,
        "width": 40,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#898B9A",
          "borderRadius": 6,
          "height": 12,
          "width": 12,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "color": "#898B9A",
        "fontFamily": "sans-serif",
        "fontSize": 14,
        "lineHeight": 22,
        "marginLeft": 8,
      }
    }
  >
    Save Me
  </Text>
</View>
`;
